SofaKernel/modules/SofaBaseLinearSolver/CMakeLists.txt

```diff
@@ -47,10 +47,10 @@ sofa_create_package_with_targets(
     INCLUDE_INSTALL_DIR "${PROJECT_NAME}"
 )
 
-# Tests
-# If SOFA_BUILD_TESTS exists and is OFF, then these tests will be auto-disabled
-cmake_dependent_option(SOFABASELINEARSOLVER_BUILD_TESTS "Compile the automatic tests" ON "SOFA_BUILD_TESTS OR NOT DEFINED SOFA_BUILD_TESTS" OFF)
-if(SOFABASELINEARSOLVER_BUILD_TESTS)
-    enable_testing()
-    add_subdirectory(${PROJECT_NAME}_test)
-endif()
+# # Tests
+# # If SOFA_BUILD_TESTS exists and is OFF, then these tests will be auto-disabled
+# cmake_dependent_option(SOFABASELINEARSOLVER_BUILD_TESTS "Compile the automatic tests" ON "SOFA_BUILD_TESTS OR NOT DEFINED SOFA_BUILD_TESTS" OFF)
```

Comment:
Commented code

Reply:
I let commented for the future(?) test in SofaBaseLinearSolver (as the move emptied all the present tests). Would you think we dont care and just delete it ?